### PR TITLE
Detect missing users while importing #4824

### PR DESCRIPTION
**Pull request by cklosowski**
#4824

### CODE REVIEW

File: includes/admin/import/class-batch-import-downloads.php [hunk 1]
```diff
@@ -106,6 +106,17 @@ public function process_step() {
 					}
 				}
 
+				if ( empty( $args['post_author'] ) ) {
+					$user = wp_get_current_user();
+					$args['post_author'] = $user->user_login;
+				} else {
+					$user = get_user_by( 'user_login', $args['post_author'] );
```
Comment (pippinsplugins) [inline includes/admin/import/class-batch-import-downloads.php]: This should detect if `post_author` has been specified with an ID or email.

Comment (cklosowski) [inline includes/admin/import/class-batch-import-downloads.php]: @pippinsplugins yeah that's an improvement. Originally our code simply worked with `user_login`, not any other fields, I was going for minimal change set.

Comment (pippinsplugins) [inline includes/admin/import/class-batch-import-downloads.php]: :+1:

**Comment by cklosowski** [comment]
Have a look at that @pippinsplugins.

**Pull request merged**